src/pm/hydra/tools/bootstrap/external/slurm_query_node_list.c

```diff
@@ -35,7 +35,7 @@ static HYD_status list_to_nodes(char *str) {
         goto fn_fail;
     }
 
-    while ((host = slurm_hostlist_shift(hostlist))) {
+    for (host = slurm_hostlist_shift(hostlist); host; host = slurm_hostlist_shirt(hostlist)) {
```

Comment:
`slurm_hostlist_shirt`? :shirt: :laughing:

Reply:
Oops. I'd fixed that and apparently forgot to push. 😄